public/src/utilityFunctions/priceUtilities.js

```diff
@@ -0,0 +1,17 @@
+//PRICE CALCULATIONS
+function calculateNights(checkOutDate, checkInDate) {
+  console.log(typeof checkOutDate, "checkout", checkInDate, "checkin")
```

Comment:
consider remove console.log if function is fully tested and working

Reply:
ah yes! will do!